martin/src/srv/tiles.rs

```diff
@@ -93,7 +93,8 @@ impl<'a> DynTileSource<'a> {
             info,
             query_str,
             query_obj,
-            encodings,
+            accept_encodings: accpect_encodings,
```

Comment:
maybe better rename these into `accept_enc` and `preferred_enc` (notice you misspelled the var there... and you are not the first - similar fate with [Referer header](https://en.wikipedia.org/wiki/HTTP_referer)) :)

One uncertainty - is this a LIST of accepted encodings, or the encoding that has been accepted (chosen)?  The name is a bit confusing - might need to make it clearer.

thx!

Reply:
I need to improve my English more XD 😂
Any better renaming suggestion? Like `accept_enc` => `browser_accept_enc`? I'm not sure...

Reply:
Or `browser_wanted_enc`?